src/home_robot/home_robot/motion/stretch.py

```diff
@@ -574,6 +581,22 @@ def _from_manip_format(self, q_raw, q_init):
         q[HelloStretchIdx.WRIST_YAW] = q_raw[6]
         return q
 
+    def _pinocchio_pose_to_ros(self, joint_angles):
+        raise NotImplementedError
+
+    def _ros_pose_to_pinocchio(self, joint_angles):
+        """utility to convert Stretch joint angle output to pinocchio joint pose format"""
+        pin_compatible_joints = np.zeros(9)
+        pin_compatible_joints[0] = joint_angles[HelloStretchIdx.BASE_X]
+        pin_compatible_joints[1] = joint_angles[HelloStretchIdx.LIFT]
+        pin_compatible_joints[2] = pin_compatible_joints[3] = pin_compatible_joints[
+            4
+        ] = pin_compatible_joints[5] = (joint_angles[HelloStretchIdx.ARM] / 4)
+        pin_compatible_joints[6] = joint_angles[HelloStretchIdx.WRIST_YAW]
+        pin_compatible_joints[7] = joint_angles[HelloStretchIdx.WRIST_PITCH]
+        pin_compatible_joints[8] = joint_angles[HelloStretchIdx.WRIST_ROLL]
+        return pin_compatible_joints
```

Comment:
We have so many of these enumerated mappings in the code. At some point we should deduplicate them and also make things less complicated.
Creating an issue from this would be nice.